src/private-key.ts

```diff
@@ -20,9 +20,9 @@ const hint = `please use:
  * @returns Private key
  * @private
  */
-export function findPrivateKey (filepath?: string): Buffer | string | null {
```

Comment:
I changed this function the `cert` option for the Probot constructor requires a `string` or `undefined` but never a Buffer.